### ISSUE TITLE
Remove Pools patch

### ISSUE THREAD

**Issue by Saibamen** [yamashi/PerformanceOverhaulCyberpunk#150]
https://www.cyberpunk.net/en/news/37166/hotfix-1-05:

`Removed the memory_pool_budgets.csv file. which was not connected with the final version of the game and had no influence on it (it was a leftover file used during the development to estimate memory usage. It had no effect on how much memory was actually allocated). Perceived performance increase after editing the file may have been related to restarting the game.`

In short: You can delete `memory_pool_budgets.csv` file and nothing will happen. We just have memory leaks in REDengine

**Comment by rmarquis** [comment]
Duplicate of #147.
